console/console-init/ui/src/modules/address-detail/AddressDetailPage.tsx

```diff
@@ -117,6 +117,8 @@ export default function AddressDetailPage() {
         addressDetail.spec.plan.metadata.name,
       planValue: addressDetail.spec.plan.metadata.name,
       topic: addressDetail.spec.topic,
+      Deadletter: addressDetail.spec.deadLetter,
```

Comment:
please revisit this name. All props will be camel cased. like `deadLetter` and `expiryQueue`